python/README.md

```diff
@@ -0,0 +1,7 @@
+# Hyperspectral Processing
+
+This package provides code for manipulating and characterizing hyperspectral imagery.
```

Comment:
Would be nice to see a small inline example here on how this library might be used

Reply:
Added a notebook and a mention of their existence in the README.  Had trouble cooking up a small example.  We'll get to that point down the line when there's more functionality in the library?